modules/dreamview/frontend/src/renderer/ground.js

```diff
@@ -23,20 +23,71 @@ export default class Ground {
             return false;
         }
 
-        const { xres, yres, mpp } = PARAMETERS.ground;
+        if (!this.render(coordinates)) {
+            return false;
+        }
+
+        this.initialized = true;
+        return true;
+    }
+
+    update(world, coordinates, scene) {
```

Comment:
is this function still useful?

Reply:
It is still called by render function of the main index